helm-chart/binderhub/values.yaml

```diff
@@ -101,7 +101,7 @@ jupyterhub:
                 # classes using traitlets?
                 # https://stackoverflow.com/questions/9575409/calling-parent-class-init-with-multiple-inheritance-whats-the-right-way
                 # https://github.com/ipython/traitlets/pull/175
-                super(BinderSpawnerMixin, self).__init__(*args, **kwargs)
+                super().__init__(*args, **kwargs)
```

Comment:
I noted this change was made by pyupgrade @manics, is it problematic do you think? So far I've not catched it make anything that is breaking. It was run configured to be compatible with py36.

Reply:
I trust the CI tests :smile: